tensorrt_llm/serve/openai_server.py

```
@@ -646,6 +678,9 @@ async def producer(generator: AsyncIterator[Any], idx: int):
             await asyncio.gather(*tasks)
 
         async def generator_wrapper(generator: AsyncIterator[Any]):
+            first_response = await anext(generator)
```

Ruff (0.13.1)

Undefined name `anext`. Consider specifying `requires-python = ">= 3.10"` or `tool.ruff.target-version = "py310"` in your `pyproject.toml` file.

(F821)